src/org/usfirst/frc/team1076/robot/subsystems/FrontBackMotors.java

```diff
@@ -27,15 +27,15 @@ public FrontBackMotors(SpeedController frontMotor, SpeedController backMotor) {
 	 * @param speed a number in the range -1 to 1 inclusive.
 	 */
 	public void setFrontSpeed(double speed) {
-		this.frontMotor.set(speed);
+		this.frontMotor.set(speed * 0.5);
```

Comment:
This seems dubious as it makes it impossible to make the motor run at full speed without passing a value of 2 to the motor. I understand why this multiplication is here, presumably it is for teleop, but if the problem is making the robot rotate and strafe at the same time, it seems better to either a) move this multiplication into the TeleopCommand class or b) create strafing and rotation methods for the drivetrain.